CMakeLists.txt

```diff
@@ -497,23 +470,15 @@ if (ISLE_BUILD_APP)
   target_link_libraries(isle PRIVATE SDL3::SDL3 iniparser-static)
 
   # Link DSOUND and WINMM
-  if(WIN32)
+  if (WIN32)
     target_link_libraries(isle PRIVATE winmm)
   endif()
   # Link LEGO1
   target_link_libraries(isle PRIVATE lego1)
-
-  target_include_directories(isle PRIVATE $<$<BOOL:${ISLE_D3DRM_FROM_WINE}>:$<TARGET_PROPERTY:d3drm-wine,INTERFACE_INCLUDE_DIRECTORIES>>)
-
-  # Make sure filenames are ALL CAPS
```

Comment:
I'm very glad to see this go, such a pain to type the names when running test builds